components/Counter/counterReducer.ts

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const counterReducer = createSlice({
+  name: 'counter',
+  initialState: { value: 0 },
+  reducers: {
+    increment(state, action) {
+      state.value -= 1;
+    },
+    decrement(state, action) {
+      state.value += 1;
```

Comment:
These are the lines that throw the ESLint error, since they modify the state parameter that is passed in. Usually, reducers should make a copy of state, modify it, and return it, but with createSlice in Redux-Toolkit, Immer sees these modifications and does the copying instead.

Reply:
I see, could you add a line to .eslintrc.json setting that rule to "off"?

Reply:
Done!